frontend/src/App.vue

```
@@ -70,10 +85,701 @@ const reportCursorDate = ref(new Date())
+  return items.filter(item => (item?.entity || '数据') === importPreviewEntityFilter.value)
+}
+
+const getFilteredPreviewItems = (items) => {
+  return applyEntityFilter(filterPreviewItems(items))
+}
+
+const previewEntities = computed(() => {
+  const details = confirmDialog.value.details
+  if (!details) return []
+  const entities = new Set()
+  ;(details.errors || []).forEach(item => entities.add(item?.entity || '数据'))
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)